pandas/core/frame.py

```diff
@@ -538,8 +538,18 @@ def _repr_html_(self):
             max_cols = get_option("display.max_columns")
             show_dimensions = get_option("display.show_dimensions")
 
-            return ('<div style="max-height:1000px;'
-                    'max-width:1500px;overflow:auto;">\n' +
+            try:
+                import IPython
+
+                if IPython.version_info < (3, 0, 0):
```

Comment:
@jreback `IPython.version_info` is a tuple. Did you mean `IPython.__version__`? Btw, why is this more preferred?